Dockerfile

```diff
@@ -1,3 +1,15 @@
 FROM nginx:alpine
-COPY ./www /usr/share/nginx/html
+
+# install required packages
+RUN apk add --no-cache git
+RUN apk add --update nodejs npm
+
+# Copy the contents and build 
+RUN mkdir /tmp/source
```

Comment:
It's unusual to use /tmp/ inside docker.
Since this thing *is* the binary-bot, I'd say `/opt/binary-bot/` normally would make sense.

Also, Minor nitpick, but normally in linux you'd have source as `src`.
But in this case, since this thing *is* the binary-bot inside the container, it can just all go in /opt/